d4m-nfs.sh

```diff
@@ -76,9 +76,8 @@ if ! $(nfsd checkexports); then
   exit 1
 else
   echo "Create the script for Moby VM"
-  # make the script for the d4m side
-  # updat
-  echo "ln -s /tmp/d4m-apk-cache /etc/apk/cache
```

Comment:
@if-kenn When you invoke this command twice, the second invocation create the wrong symlink. `ln -ns` fix the problem.